modules/web-element.mjs

```diff
@@ -121,9 +125,11 @@ class DimensionWidth extends WebElementFunction {
         super("width");
     }
 
-    get(e) {
-        var s = e.style.width;
-        return parseFloat(s);
+    get(element) {
+        //var doc = element.ownerDocument;
+        //var win = doc.defaultView || doc.parentWindow;
+        var win = this.GetOwnerWindow(element)
+        return parseFloat(win.getComputedStyle(element).getPropertyValue('width'))
```

Comment:
Maybe a utility method should be made to get an element's computed styles, too, seeing as it's currently being used in many methods.

Ex.:
```js
getElementComputedStyle(element) {
    const window = this.getOwnerWindow(element);
    return window.getComputedStyle(element);
}
```

Reply:
Actually, as every function that fetches the value of a CSS property is the same, should an abstract function class be made and reused for all of them? 

Ex.: 
```js
class CssPropertyFunction extends  WebElementFunction
{
     get(element)
     {
        const style = this.getComputedStyle(element);
        return style.getPropertyValue(this.name);
     }
}
```

and then specific functions could look like: 
```js
class Color extends CssPropertyFunction {
    constructor(){
        super("color")
    }
}
```

Maybe @sylvainhalle can give his two cents on this...

Reply:
Agree! The method should be declared `static`.